client/my-sites/plan-features/docs/example.jsx

```diff
@@ -12,43 +12,94 @@ import PlanFeaturesItem from '../item';
 import PlanFeaturesItemList from '../list';
 import PlanFeaturesFooter from '../footer';
 import { plansList, PLAN_FREE, PLAN_PREMIUM, PLAN_BUSINESS } from 'lib/plans/constants';
+import SelectDropdown from 'components/select-dropdown';
+
+const options = [
+	{ value: 'USD', label: 'USD' },
+	{ value: 'AUD', label: 'AUD' },
+	{ value: 'CAD', label: 'CAD' },
+	{ value: 'EUR', label: 'EUR' },
+	{ value: 'GBP', label: 'GBP' },
+	{ value: 'JPY', label: 'JPY' }
+];
+
+const currencyData = {
+	USD: {
+		currencyCode: 'USD',
+		free: 0,
+		premium: 99 / 12,
+		business: 299 / 12
+	},
+	AUD: {
+		currencyCode: 'AUD',
+		free: 0,
+		premium: 129 / 12,
+		business: 399 / 12
+	},
+	CAD: {
+		currencyCode: 'CAD',
+		free: 0,
+		premium: 129 / 12,
+		business: 389 / 12
+	},
+	EUR: {
+		currencyCode: 'EUR',
+		free: 0,
+		premium: 99 / 12,
+		business: 299 / 12
+	},
+	GBP: {
+		currencyCode: 'GBP',
+		free: 0,
+		premium: 85 / 12,
+		business: 250 / 12
+	},
+	JPY: {
+		currencyCode: 'JPY',
+		free: 0,
+		premium: 11800 / 12,
+		business: 35800 / 12
+	}
+};
 
 export default React.createClass( {
 
 	displayName: 'PlanFeatures',
 
 	mixins: [ PureRenderMixin ],
 
-	render() {
-		const priceFree = {
-			currencySymbol: '$',
-			decimalMark: '.',
-			dollars: 0,
-			cents: 0
-		};
-		const pricePremium = {
-			currencySymbol: '$',
-			decimalMark: '.',
-			dollars: 8,
-			cents: 25
-		};
-		const priceBusiness = {
-			currencySymbol: '$',
-			decimalMark: '.',
-			dollars: 24,
-			cents: 92
+	onDropdownSelect( selected ) {
+		this.setState( { currency: selected.value } );
+	},
+
+	getInitialState() {
+		return {
+			currency: 'USD'
 		};
+	},
+
+	render() {
+		const priceData = currencyData[ this.state.currency ];
+
 		return (
 			<div className="design-assets__group">
 				<h2>
 					<a href="/devdocs/app-components/plan-features">Plan Features</a>
+
 				</h2>
 				<div>
+					<SelectDropdown
+						options={ options }
+						onSelect={ this.onDropdownSelect } />
+					<br />
+				</div>
+				<div>
 					<PlanFeaturesHeader
 						current
 						title={ plansList[ PLAN_FREE ].getTitle() }
 						planType={ PLAN_FREE }
-						price={ priceFree }
+						rawPrice={ priceData.free }
```

Comment:
It won't work here like this because `<PlanFeaturesHeader>` is now a part of `<PlanFeatures>` component which is connected to the state tree. However, we could probably add some special case for devdocs. However, I'm not sure if it is worth the effort.

On the other hand, it would be nice to be able to test `PlanFeatures` with different currencies.